workbench-core/example/app/src/routes/auditRoutes.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuditService, Metadata, MetadataParser } from '@aws/workbench-core-audit';
+import { isAuditIncompleteError } from '@aws/workbench-core-audit/lib/errors/auditIncompleteError';
+import { DynamicAuthorizationService } from '@aws/workbench-core-authorization';
+import { validateAndParse } from '@aws/workbench-core-base';
+import * as Boom from '@hapi/boom';
+import { Request, Response, Router } from 'express';
+import { wrapAuth } from '../utilities/authWrapper';
+
+export function setupAuditRoutes(
+  router: Router,
+  auditService: AuditService,
+  dynamicAuthorizationService: DynamicAuthorizationService
+): void {
+  router.post(
+    '/audit/write',
+    wrapAuth(dynamicAuthorizationService, async (req: Request, res: Response) => {
+      try {
+        const authenticatedUser = res.locals.user;
+        const source = {
+          ip: req.headers['x-forwarded-for'] || req.socket.remoteAddress
+        };
+        const statusCode: number = parseInt(req.body.statusCode, 10);
+        const metadata = {
+          actor: authenticatedUser,
+          source,
+          action: req.body.action,
+          statusCode
+        };
+        const body = req.body.responseBody;
+        const validatedMetadata = validateAndParse<Metadata>(MetadataParser, metadata);
+        await auditService.write(validatedMetadata, body);
+        res.status(200).send();
+      } catch (err) {
+        if (isAuditIncompleteError(err)) {
+          throw Boom.badRequest(err.message);
+        }
+        throw err;
+      }
+    })
+  );
+
+  router.post(
```

Comment:
I probably would have made this a `router.get()`

Reply:
fixed